tox.ini

```diff
@@ -35,10 +55,14 @@ usedevelop = True
 
 [testenv:metadata-validation]
 deps =
+  {[testenv:build-dists]deps}
   collective.checkdocs
   twine
 usedevelop = False
+skip_install = true
 # Ref: https://twitter.com/di_codes/status/1044358639081975813
 commands =
-  python -m setup checkdocs check --metadata --restructuredtext --strict --verbose
-  twine check .tox/dist/*
+  {[testenv:build-dists]commands}
```

Comment:
No, we don't need to embed building here. It should stay decoupled.

Reply:
Not adding this here would be a serious bug, it would mean that we check a potentially empty folder, or an outdated one. Running tox on any environment should never made the assumption that the user already run successfully one of the other environments.

Tox environment executions are supposed to be predictable and easy to reproduce, not to depend on previous executions.